Documentation/commands.md

```diff
@@ -395,3 +395,56 @@ Garbage collecting pod "21b1cb32-c156-4d26-82ae-eda1ab60f595"
 Garbage collecting pod "5dd42e9c-7413-49a9-9113-c2a8327d08ab"
 Garbage collecting pod "f07a4070-79a9-4db0-ae65-a090c9c393a3"
 ```
+
+### rkt image export
+
+There are cases where you might want to export the ACI from the store to copy to another machine, file server, etc.
+
+```
+$ rkt image export coreos.com/etcd etcd.aci
+$ tar xzvf etcd.aci
+```
```

Comment:
```
$ tar xzvf etcd.aci 
gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

The tar option "z" does not work because it is not compressed:
```
$ file etcd.aci 
etcd.aci: POSIX tar archive
```